src/Microsoft.AspNet.Razor/Chunks/ChunkTreeBuilder.cs

```diff
@@ -27,13 +27,13 @@ public void AddChunk(Chunk chunk, SyntaxTreeNode association, bool topLevel = fa
             chunk.Association = association;
 
             // If we're not in the middle of a chunk block
```

Comment:
This was the only comment issue I caught but it may be worth doing global searches for "chunk block" and "block code".